src/librustc_mir/interpret/validity.rs

```diff
@@ -776,14 +781,14 @@ impl<'rt, 'mir, 'tcx, M: Machine<'mir, 'tcx>> ValueVisitor<'mir, 'tcx, M>
                         // For some errors we might be able to provide extra information
                         match err.kind {
                             err_ub!(InvalidUndefBytes(Some(ptr))) => {
-                                // Some byte was undefined, determine which
+                                // Some byte was uninitialized, determine which
                                 // element that byte belongs to so we can
                                 // provide an index.
                                 let i = usize::try_from(ptr.offset.bytes() / layout.size.bytes())
                                     .unwrap();
                                 self.path.push(PathElem::ArrayElem(i));
 
-                                throw_validation_failure!("undefined bytes", self.path)
+                                throw_validation_failure!("uninitialized bytes", self.path)
```

Comment:
Now I'm confused. I understand that values can't be undefined but unintialized, but bytes/memory can still be undefined, right?

Reply:
No I don't think so -- memory can certainly be uninitialized, and I think you mean the same thin when you say "undefined", so we should stick to the terminology we also use elsewhere.

Reply:
ok